jrpc/internal/service.go

```diff
@@ -286,7 +286,10 @@ func (s *jasperService) ResetTags(ctx context.Context, id *JasperProcessID) (*Op
 func (s *jasperService) DownloadMongoDB(ctx context.Context, opts *MongoDBDownloadOptions) (*OperationOutcome, error) {
 	jopts := opts.Export()
 	if err := jopts.Validate(); err != nil {
-		return &OperationOutcome{Success: false, Text: errors.Wrap(err, "problem validating MongoDB download options").Error()}, errors.Wrap(err, "problem validating MongoDB download options")
+		return &OperationOutcome{
+			Success: false,
+			Text:    errors.Wrap(err, "problem validating MongoDB download options").Error(),
+		}, errors.Wrap(err, "problem validating MongoDB download options")
```

Comment:
should this be nil error?

Reply:
Yup this is what I meant by "It may be worth it to do something similar to other times where we return (*OperationOutcome, error) and its better to send over the *OperationOutcome instead.".

I think we could just include this in this PR itself if we wanted. I'll go ahead and do this.

Ditto your other question.